.deprecated/collection_processor.py

```diff
@@ -145,13 +141,17 @@ def post_collection(self, stac_host, json_data):
 
         Returns the collection id.
         """
-        collection_id = json_data['id']
+        collection_id = json_data["id"]
         r = requests.post(os.path.join(stac_host, "collections"), json=json_data, verify=False)
 
         if r.status_code == 200:
-            print(f"{bcolors.OKGREEN}[INFO] Pushed STAC collection [{collection_id}] to [{stac_host}] ({r.status_code}){bcolors.ENDC}")
+            print(
+                f"{bcolors.OKGREEN}[INFO] Pushed STAC collection [{collection_id}] to [{stac_host}] ({r.status_code}){bcolors.ENDC}"
+            )
         elif r.status_code == 409:
-            print(f"{bcolors.WARNING}[INFO] STAC collection [{collection_id}] already exists on [{stac_host}] ({r.status_code}), updating..{bcolors.ENDC}")
+            print(
```

Comment:
Use python's warnings (https://docs.python.org/3/library/warnings.html) to display warnings to the user.

Reply:
Deprecated code